exercises/bloco_7/dia_3/bonus1.js

```diff
@@ -0,0 +1,48 @@
+const assert = require('assert');
+
+function getChange(payable, paid) {
+  const coins = [200, 100, 50, 20, 10, 5, 2, 1];
+  const change = [];
+  const { length } = coins;
+  let remaining = paid - payable;
+
+  // escreva seu código aqui...
+  if (payable > paid) throw new Error('paid value is not enough');
+  if (payable === paid) return [];
+
+
+//  Adicionando a função mais criminosa do universo 
+  let arr = [];
+  for (let i = 0; i < remaining; i) {
+    if (remaining >= 200) {arr.push(200); remaining -= 200;}
```

Comment:
Olá rodrigo, meu parça! Beleza?
O último argumento do for é algo que vai ser rodado antes de começar novamente.
Um chamado de `i` não faz nada. O que acha de remover ele? :wink: Você pode terminar seu for assim:

`for (let i = 0; i < remaining;)`

Reply:
Boa Inácio, o Bernardo tinha me falado isso no plantão hoje quando mostrei a função pra ele, mas tinha até me esquecido, valeu!